server/superdesk/data_consistency/compare_repositories_tests.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import superdesk
+from eve.utils import ParsedRequest
+from superdesk.tests import TestCase
+from superdesk import get_resource_service
+from superdesk.data_consistency.compare_repositories import CompareRepositories
```

Comment:
I think it is a good idea to come-up with a generic helper class like this where we can ask for ClassUnderTest to get loaded. I think we should do this across the board. Maybe we can create a technical ticket for this suggestion and one of us can pick it up. I'll probably merge this PR with its current shape now as we need to see this working. Thanks Peter.